fellow-submissions/Cielo-Raymundo.md

```diff
@@ -0,0 +1,138 @@
+    ```javascript
+    function hello() {
+      let a = 'hello';
+    }
+
+    hello();
+    console.log(a);
+    ```
+    This will console log:
+    undefined
+    a is undefined
+    
+    Because the function isn't returning anything so it's just undefined. Since the variable a only exists inside the function hello, it can't 
+    be accessed outside the function/scope. 
+**4. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    var a = 1;
+    ```
+   This will console log:
+   Undefined
+   
+   Because even though the variable is being hoisted, only the declaration is being read so it isn't initialized until it is fully compiled. 
+   
+**5. What does the following code log? Why?**
+    ```javascript
+    console.log(a);
+
+    let a = 1;
+    ```
+    This will console log:
+    a is undefined
+    
+    Because let variables don't get hoisted, they don't exist before they are declared. So the variable a is something that doesn't exist
+    
+**6. In the code snippet below, the function declaration comes _after_ the variable declaration in the code below. Why does it still work?**
+    ```javascript
+    let name = "Devonte";
+
+    function sayItLoud() {
+      console.log(`${name}!!!`)
+    }
+
+    sayItLoud();
+    ```
+    Because name is a variable who's blockscope includes the function sayItLoud, so when that function uses the variable name, it will
+    take the value it was assigned to.
+    
+**7. Why does the following code throw an error? What is the key difference between it and the example above?**
+    ```javascript
+    sayItLoud();
+
+    function sayItLoud() {
+      console.log(`${greeting}!!!}`)
+    }
+
+    let greeting = "Hello"
+    ```
+    This doesn't work because the function sayItLoud doesn't exist inside the variables greeting's scope so greeting inside the function doesn't
+    exist.
+
+**8. What does the following code log? Why?**
+    ```javascript
+    let problemSet;
+
+    function complete(assignment) {
+      assignment = "done";
+    }
+
+    complete(problemSet);
+    console.log(`Status for Problem Set: ${problemSet}`);
+    ```
+    This will console log:
+    undefined
+    
+    Because the variable problemSet is declared but not initialized, and it's being console logged in it's scope so it exists but has no value.
+
+**9. Why does this code throw an error?** 
+    ```javascript
+    const isSunny🌞 = true;
+
+    if (isSunny🌞) {
+      let advice = 'Wear your sunshades!';
+    }
+
+    console.log(advice);
+    ```
+    This throws an error because the variable advice doesn't exist outside of the if function since it can't be hoisted.
+
+**10. So... why does this code work as intended?**
+    ```javascript
+    const isRainy🌧 = true;
+
+    if (isRainy🌧) {
+      var advice = 'Pack your umbrella.';
+    }
+
+    console.log(advice);
+    ```
+    This works because the variable advice gets hoisted so it exists before the variable is even declared. 
```

Comment:
This doesn't have to do with hoisting because we define the variable before we log it. This has to do with _scoping_. Are variables declared with `var` block scoped?